src/toolkits/object_detection/od_data_iterator.cpp

```diff
@@ -304,6 +309,13 @@ simple_data_iterator::compute_properties(
     result.classes = std::move(expected_class_labels);
     int i = 0;
     for (const std::string& label : result.classes) {
+      if (classes_inferred.find(label) == classes_inferred.end() &&
```

Comment:
Move the if(is_training_) around the entire for loop for clarity.

Reply:
good!